transformer_engine/pytorch/attention.py

```diff
@@ -1056,8 +1060,10 @@ def apply_rotary_pos_emb(t: torch.Tensor, freqs: torch.Tensor, tensor_format: st
 
     # Only apply the rotary embeddings up to the sequence length of the running
     # input.
-    assert cur_seq_len <= max_seq_len, (f"Rotary Embeddings only supported "
```

Comment:
Why?

Reply:
I changed the `assert error_cond,  err_msg` to `if (error_cond): raise Exception(err_msg)` to make it consistent with rest of the code.
(I was gonna change this line anyways since lint wasn't happy:  `W1309: Using an f-string that does not have any interpolated variables (f-string-without-interpolation)`)